modules/installation-configuration-parameters.adoc

```diff
@@ -1267,6 +1267,10 @@ Optional VMware vSphere machine pool configuration parameters are described in t
 |`platform.vsphere.memoryMB`
 |The size of a virtual machine's memory in megabytes.
 |Integer
+
+|`platform.vsphere.disk_type`
```

Comment:
here should be `platform.vsphere.diskType`

Reply:
Correction has been made